### DISCUSSION TITLE
Lock Page for not editing page accidently

### DISCUSSION THREAD

**Discussion by kesuskim** [outline/outline#4039] [Ideas]
Just like notion does, it would be great to have feature 'lock page'.

![image](https://user-images.githubusercontent.com/7637982/187618245-6ad401cf-2788-41a2-81d7-af981ad341ae.png)

Related:
https://github.com/outline/outline/discussions/3311#discussioncomment-2478670

**Comment by tommoor** [comment]
Also #3132

**Reply by kesuskim** [reply]
Oh yes, looking forward to keyword 'lock page' yet failed, there is it 😄

**Reply by arsen-z** [reply]
I think it is certainly related, but quite a different issue, though, because in Notion it is possible to lock a single page, whereas the "Edit" button could be enabled either for all pages or for none at all.

**Comment by whihathac** [comment]
Would totally love this feature!
